### PR TITLE
Add diversity page

### PR DESCRIPTION
**Pull request by dlsniper**
To be discussed before merging.

### CODE REVIEW

File: diversity.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Diversity
+permalink: /diversity/
+isStaticPost: true
+---
+
+##### DevFest Berlin supports diversity in technology
+
+We recognize the current status of our industry and we want to want to change it.
```
Comment (mariobodemann) [inline diversity.md]: `want to` twice (unless we only want to want and not actually change it ... ;))

File: diversity.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+To this end, we have have a couple of ways to support our mission:
+
+- we will have a number of tickets for which only underrepresented groups can apply
+- we will have a mentorship program for new programmers
+
+The mentorship program will pair developers willing to share their passion and
+knowledge of technology with new developers or persons interested in becoming a
+developer. We believe this will yield a great value both to the mentor and the
+mentored person and hope every everyone will have a great opportunity to learn
```
Comment (mariobodemann) [inline diversity.md]: `every everyone` ;)

File: diversity.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+We recognize the current status of our industry and we want to want to change it.
+
+Our goal is to enable everybody to learn and teach programming. As such, we
+want to enable underrepresented groups in technology to feel welcomed and help
+increase the diversity in technology and science.
+
+To this end, we have have a couple of ways to support our mission:
+
+- we will have a number of tickets for which only underrepresented groups can apply
+- we will have a mentorship program for new programmers
```
Comment (Pisush) [inline diversity.md]: i like `for new developers`, as used below, better

File: diversity.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+The mentorship program will pair developers willing to share their passion and
```
Comment (Pisush) [inline diversity.md]: What is actually happening at the mentorship program is not clear if you are not familiar with the concept. How about -

`The mentorship program will pair an awardee with a developer mentor to help familiarize the awardee with the conference. We believe creating a dialogue between developers at different levels will yield a great value both to the mentor and the awardee and hope everyone will have a great opportunity to learn from it.`

**Pull request merged**